File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/PasportDataFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.passport.PassportData;

public class PasportDataFilterValidator extends FilterValidator<PassportData> {

    public PasportDataFilterValidator(CustomValidator<PassportData> validator) {
        super(PassportData.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/PasportDataFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.passport.PassportData
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class PasportDataFilterValidator(validator: Validator<PassportData>) : FilterValidator<PassportData>(
    PassportData::class, validator)
